examples/sanity/README.md

```diff
@@ -0,0 +1,66 @@
+# Sanity Example!
```

Comment:
Could you update this to follow the format in the `examples/template/README.md` please?